R/data.table.R

```diff
@@ -12,9 +12,11 @@ methods::setPackageName("data.table",.global)
 #   (2) export() in NAMESPACE
 #   (3) add to vignettes/datatable-importing.Rmd#globals section
 .SD = .N = .I = .GRP = .NGRP = .BY = .EACHI = NULL
+. = function(...) NULL
+J = function(...) NULL
 # These are exported to prevent NOTEs from R CMD check, and checkUsage via compiler.
 # But also exporting them makes it clear (to users and other packages) that data.table uses these as symbols.
-# And NULL makes it clear (to the R's mask check on loading) that they're variables not functions.
+# The use of NULL during loading signals to R's mask check that these entities aren't defined with specific behaviors.
```

Comment:
why "during loading" ?? I find it confusing, please delete unless you have a strong reason to keep. 
why "aren't defined with specific behaviors" ?? each of these actually does have a specific behavior. please either delete or clarify.

Reply:
Yeah lets delete that, to begin with this comment was their to notify that NULL signals that they're variables not functions, but now that we have function in same condition.

> why "during loading" ?

Same as the previous comment, `makes it clear (to the R's mask check on loading) `

>why "aren't defined with specific behaviors" ?

Yeah this seems to be a mistake by me, tbf i don't clearly remember what was going on when i wrote this but this seems wrong now.

Thanks for suggestions :)